webapp/components/pokemon-table.component.ts

```diff
@@ -20,13 +20,41 @@ export class PokemonTableComponent {
 	//right now we're only allowing 1 transfer at a time, like God intended,
 	//this may need to be updated later to allow for batch transfers
 	private _transferringPokemonAtIndex: number = null;
+	private _renamingPokemonAtIndex: number = null;
 	private _currentSortOrderName: string = '';
 
-	constructor(private _properties: PropertiesService, private _pokemonService: PokemonService) { 
+	constructor(private _properties: PropertiesService, private _pokemonService: PokemonService) {
 	}
 
-	public set pokemon(pokemon: Pokemon[]){
-		this._pokemon = pokemon;
+	public set pokemon(pokemons: Pokemon[]){
```

Comment:
because it's an array, and a few lines under I do something like this : 
```js
pokemons.forEach(function (pokemon) {
  ...
});
```

It's an habit I have to name arrays with a plural, I can change it if you think it doesn't fit the code style of the project.